event/register_oracle_event.go

```diff
@@ -51,9 +59,93 @@ func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
 
 	err = sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, nodePubKeyHash[:], *e.reactor.EnclaveInfo())
 	if err != nil {
+		msgVoteOracleRegistrationNo, err := makeOracleRegistrationVote(uniqueID, e.reactor.OracleAcc().GetAddress(), addressValue, types.VOTE_OPTION_NO, e.reactor.OraclePrivKey())
+		if err != nil {
+			return err
+		}
+
+		txBytes, err := generateTxBytes(msgVoteOracleRegistrationNo, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), e.reactor.TxBuilder())
+		if err != nil {
+			return err
+		}
+
+		err = broadCastTx(e.reactor.GRPCClient(), txBytes)
+		if err != nil {
+			return err
+		}
```

Comment:
These codes are duplicated. I think we can share this code block with the case of `YES`

Reply:
Since this error occurred while executing `generateTxBytes()` or `broadCastTx()`, wouldn't it be appropriate to return an error?